src/tools/ReadyToRun.SuperIlc/ParallelRunner.cs

```diff
@@ -270,7 +270,7 @@ private static void BuildProjects(int startIndex, int endIndex, int totalCount,
                 }
                 while (freeSlot == null);
 
-                freeSlot.Launch(processInfo, jittedMethods, index, totalCount, progressIndex, failureCount);
+                freeSlot.Launch(processInfo, jittedMethods, index + 1, totalCount, progressIndex, failureCount);
```

Comment:
> + 1 [](start = 65, length = 4)

What's the +1 for?

Reply:
See: https://github.com/dotnet/coreclr/pull/26742#discussion_r325428392
I think I accidentally removed that fix in a later commit in that PR.